lib/oidcstrategy.js

```diff
@@ -144,42 +151,114 @@ function onProfileLoaded(strategy, args) {
  *
  * Options:
  *
- *   - `scope`              list of scope values indicating the required scope of the
- *                          access token for accessing the requested resource. Ex: ['email', 'profile']. 
- *                          We send 'openid' by default and will reject you if you send it a second time.
- *   - `audience`           we check JWT audience (aud), provide a value here
- *   - `clientID`           the application ID of your app in Microsoft Identity platform
- *   - `identityMetadata`   the metadata endpoint provided by the Microsoft Identity Portal that provides 
- *                          the keys and other important info at runtime. We roll keys frequently, so don't
- *                          override this to supply your own.
- *   - `responseType`       for login only flows use id_token. For accessing resources use `id_token code`
- *   - `responseMode`       For login only flows we should have token passed back to us in a POST
- *   - `validateIssuer`     if you have validation on, you cannot have users from multiple tenants sign in
- *   - `passReqToCallback`  if you want the Req to go back to the calling function for other processing use this.
- *   - `nonceLifetime`      the lifetime of nonce in session, default is NONCE_LIFE_TIME
+ *   - `identityMetadata`   (1) Required
+ *                          (2) must be a https url string
+ *                          (3) Description:
+ *                          the metadata endpoint provided by the Microsoft Identity Portal that provides 
+ *                          the keys and other important info at runtime. Examples:
+ *                          <1> v1 non-common endpoint
```

Comment:
Agree with Brent. We should call this "v1 tenant-specific (not common) endpoint".